feature-based-molecular-networking/tools/feature-based-molecular-networking/scripts/progenesis_formatter.py

```diff
@@ -7,26 +7,41 @@
 import pandas as pd
 import sys
 
-
-#TODO: Ask about why there are two column headers of the same name
 def convert_to_feature_csv(input_filename, output_filename):
-    input_format = pd.read_csv(input_filename, sep=",", skiprows=2)
+    # First read the table and deduce the number of samples from the difference between Norm abundance and RAW abundance (skiprows=0)
```

Comment:
What is the implicit assumption here for ordering?

Reply:
This makes possible to deduce the number of samples regardless of the number of sample grouping columns.

Reply:
Can you clarify above with an example file showing what you mean in the README the format. Its still unclear how you differentiate in the documentation a sample vs metadta.

Reply:
yes